app/auth/mutations/login.ts

```diff
@@ -17,7 +17,7 @@ export const authenticateUser = async (rawEmail: string, rawPassword: string) =>
     await db.user.update({ where: { id: user.id }, data: { hashedPassword: improvedHash } })
   }
 
-  const { hashedPassword, ...rest } = user
+  const { ...rest } = user
```

Comment:
It doesn't seem right:
1. Why we are destructuring the user if we return the entire result anyway.
2. The previous intention (to not include the password in the response) has been changed without a good reason.
If this change is a result of a linter error, you should use the omit function from lodash or some equivalent. As a result, you will return the user but without the `hashadPassword` property.

Reply:
This is a linter error about assigned value and never used it. By omit function you mean to disable
eslint for this line?